test/benchmark/test/only.bench.ts

```diff
@@ -0,0 +1,59 @@
+import { bench, describe, expect } from 'vitest'
```

Comment:
This file is still not running, I just put the previous comment on the first line. Please, ensure that your tests are passing before requiring review.

Reply:
I see, I didn't know these benchmarks are not running, so misunderstood you! Let me see.

Reply:
I see, I didn't know these benchmarks are not running, so misunderstood you! Let me see.